tests/test_web_helpers.py

```diff
@@ -0,0 +1,160 @@
+import json
+import unittest
+from django.conf import settings
+
+from core_lib.helpers.constants import MediaType
+from core_lib.web_helpers.request_response_helpers import response_status, response_ok, response_message, \
+    response_download_content, response_json
+from core_lib.web_helpers.web_helprs_utils import WebHelpersUtils
+
+
+class TestWebHelpers(unittest.TestCase):
+
+    def test_web_utils_flask(self):
+        web_util = WebHelpersUtils()
+
+        web_util.init("flask")

```

Comment:
this is thje proper way to initilalize it 
`WebHelpersUtils.init(WebHelpersUtils.ServerType.DJANGO)`